File: 58-length-of-last-word/length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        x=list(s.split(" "))
        y=[]
        for i in x:
            if i:
                y.append(i)
        return len(y[-1])

File: 58-length-of-last-word/length-of-last-word.java
// class Solution {
//     public int lengthOfLastWord(String s) {
//         String word[] = s.split(" ");
//         return word[word.length-1].length();
//     }
// }
class Solution {
    public int lengthOfLastWord(String s) {
        int len = 0;
        int i=s.length()-1;
        while(i>=0){
            if(s.charAt(i)==' '){
                i--;
            }
            else{
                break;
            }
        }

        while(i>=0 && s.charAt(i)!=' '){
            len++;
            i--;
        }
        return len;
    }
}